docs/advanced-features/workflows.md

```diff
@@ -354,6 +354,59 @@ With this Schema lists such as
 `["Fixed 1", "Any string", 123]` 
 are valid but `["Any string", 123]` are invalid.
 
+### Example properties
+The following examples show how you can specify complex properties
+such as `object` and `array` data types with arbitrary nesting.
+
```

Comment:
Looks good please add back `#### Objects section` no explanation and we're good.

Reply:
`#### Objects` section re-added (code snippet) and no explanation